pkg/reconciler/mongodb/mongodbsource.go

```diff
@@ -26,31 +28,42 @@ import (
 
 	v1alpha1 "github.com/googleinterns/knative-source-mongodb/pkg/apis/sources/v1alpha1"
 	mongodbsource "github.com/googleinterns/knative-source-mongodb/pkg/client/injection/reconciler/sources/v1alpha1/mongodbsource"
+	"knative.dev/pkg/apis"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+
 
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
-	v1 "k8s.io/api/core/v1"
+
+	corev1 "k8s.io/api/core/v1"
 	corev1listers "k8s.io/client-go/listers/core/v1"
 	reconciler "knative.dev/pkg/reconciler"
+	"knative.dev/pkg/resolver"
 )
 
 // newReconciledNormal makes a new reconciler event with event type Normal, and
 // reason MongoDbSourceReconciled.
 func newReconciledNormal(namespace, name string) reconciler.Event {
-	return reconciler.NewEvent(v1.EventTypeNormal, "MongoDbSourceReconciled", "MongoDbSource reconciled: \"%s/%s\"", namespace, name)
+	return reconciler.NewEvent(corev1.EventTypeNormal, "MongoDbSourceReconciled", "MongoDbSource reconciled: \"%s/%s\"", namespace, name)
+
 }
 
 // Reconciler implements controller.Reconciler for MongoDbSource resources.
 type Reconciler struct {
 	// Lister
 	secretLister corev1listers.SecretLister
+
+	sinkResolver *resolver.URIResolver
+
```

Comment:
nit: no need for this extra empty line

Reply:
I dont know why a lot of these extra lines dont exist on my local files??